python/postprocessing/modules/common/lepSFProducer_v2.py

```diff
@@ -0,0 +1,140 @@
+import ROOT
```

Comment:
isn't lepSFProducer_v2 factorized out in another PR? are there cross dependencies witht he rocc?

Reply:
There should not be any dependencies. I have not made the PR for lepSFProducer_v2.